code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -41,28 +46,78 @@
 	eyeobj.icon = 'icons/obj/abductor.dmi'
 	eyeobj.icon_state = "camera_target"
 
+/obj/machinery/computer.camera_advanced/xenobio/New()
```

Comment:
not calling parent is very dangerous for `New`, especially for machines. Don't do it.